logging/setup.py

```diff
@@ -50,13 +50,13 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.21.0, < 0.22dev',
+    'google-cloud-core >= 0.22.1, < 0.23dev',
     'gapic-google-cloud-logging-v2 >= 0.14.0, < 0.15dev',
```

Comment:
This should be updated to `>= 0.90.0, < 0.91dev` per https://github.com/googleapis/api-client-staging/pull/137/.

The other GAPIC dependencies will remain at 0.14.x, since they are not yet Beta.